drivers/video/Kconfig.hm01b0

```diff
@@ -1,6 +1,6 @@
 # Copyright The Zephyr Project Contributors
 # SPDX-License-Identifier: Apache-2.0
-config VIDEO_HIMAX_HM01B0
+config VIDEO_HM01B0
```

Comment:
Good catch thank you!

You might remove the `not maybe necessary,` from the commit if wanting a bit more canonical style, where discussion about "is this the correct thing to do" is left outside the commit message.

Likewise "so will test this out" is perfect for the PR discussion thread, but in the commit message, best to only have description of the changes... The same content as a big `/* code comment */` describing the patch.

In an email, there would be first a textual comment in prose *before* the commit messages then the patches with their short commit message.

Just a matter of conventions.